File: noi.openjudge.cn/ch0103/07.cpp
// http://noi.openjudge.cn/ch0103/solution/38720779/

#include<iostream>
using namespace std;
int main() {
    double a,b,c,d,x;
    cin >> x >> a >> b >> c >> d;
    printf("%0.7lf\n",(a*x*x*x)+(b*x*x)+(c*x)+d);
}

File: noi.openjudge.cn/ch0108/07.py
# http://noi.openjudge.cn/ch0109/solution/39961778/

n = int(input())
a = list(map(int,input().split()))
m = 0
b = 0
for x in a:
    if x > m:
        m = x
for x in a:
    if x != m:
        b += x
print(b)
